acceptance/tests/utf8/utf8-in-puppet-describe.rb

```diff
@@ -0,0 +1,71 @@
+test_name 'utf-8 characters in module doc string, puppet describe' do
+  # utf8chars = "€‰ㄘ万竹ÜÖ"
+  utf8chars = "\u20ac\u2030\u3118\u4e07\u7af9\u00dc\u00d6"
+  agents.each do |agent|
+    mod_dir = agent.tmpdir("describe")
```

Comment:
i disagree. i tend to leave these around as they are tmpdirs, created uniquely(-ish) per run.  leaving them around aids in test debugging, where required.  

but i wouldn't hold up the merge, either way